common/src/stateTypes.ts

```diff
@@ -0,0 +1,54 @@
+import { VisibleOptions } from './options';
+import { z } from 'zod';
+
+export type Player = {
+  id: string;
+  name: string;
+  leader: boolean;
+  active: boolean;
+};
+
+export type Stage = 'lobby' | 'response' | 'selection' | 'matching' | 'endRound';
```

Comment:
@kajgrant note the Stage type isn't a string, it's a union of string values. There's some weird reasons that string unions tend to be preferred over const enums in typescript.